### PR TITLE
Msdk 743 update to 4.1.0 types

### PR DESCRIPTION
**Pull request by polymath-eric**

### CODE REVIEW

File: src/polkadot/definitions.ts [hunk 1]
```diff
@@ -1,5 +1,3 @@
-/* istanbul ignore file */
-
```
Comment (VictorVicente) [inline src/polkadot/definitions.ts:2]: This file must be ignored, please restore this line

**Review by VictorVicente — CHANGES_REQUESTED** [review]
Hey @polymath-eric, well done.
I think you need to fix the test case for the versioning control. That's the reason why all tests are failing.

**Review by monitz87 — APPROVED** [review]

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**